spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_task do
+      status { 'done' }
```

Comment:
enumで設定したIntegerのカラムに対する値なので、StringよりSymbolを使った方がカラムの種類が伝わりますね。
```suggestion
      status { :done }
```

Reply:
7600b17d21bb0d6c862f909bdd15075d4704f0c8 
で修正しました